### PR TITLE
Introduced PR templates

### LINKED ISSUE #786: request: add PR template
add a PR template that contributors will use to check their work against the accepted standards of the project their are contributing to

### PR DESCRIPTION
**Pull request by koconder**
Fixes #786

Based on main repo PR template, see: https://github.com/hasura/graphql-engine/blob/master/.github/PULL_REQUEST_TEMPLATE.md

### CODE REVIEW

**Comment by koconder** [comment]
@NoriSte @andoks @praveenweb quick one to close #786

**Comment by NoriSte** [comment]
I'd like to get a review from @praveenweb since he has more context about the whole product (since I have comprehension for the Console part only).

And, of course, thank you so much @koconder 😊 I'll send it to Praveen directly 😊

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<!-- Thank you for submitting this PR! :) -->
+<!-- Provide a general summary of your changes in the Title above ^, end with (close #<issue-no>) or (fix #<issue-no>) -->
+
+### Description
+<!-- The title might not be enough to convey how this change affects the user. -->
+<!-- Describe the changes from a user's perspective -->
+
+### Changelog
```
Comment (praveenweb) [inline PULL_REQUEST_TEMPLATE.md:8]: In the learn course template, we don't have versions or changelog. Can we remove this section?

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+### Steps to test and verify
+<!-- If this is a feature, what are the steps to try them out? -->
+<!-- If this is a bug-fix, how do we verify the fix? -->
+
+### Limitations, known bugs & workarounds
+<!-- Limitations of the PR, known bugs and suggested workarounds -->
+<!-- Feel free to delete these comment lines -->
+
+#### Breaking changes
```
Comment (praveenweb) [inline PULL_REQUEST_TEMPLATE.md:47]: Similarly, there are no breaking changes applicable.

**Review by praveenweb — CHANGES_REQUESTED** [review]
Added comments to remove a couple of sections

**Comment by koconder** [comment]
@praveenweb changes made as requested

**Pull request merged**